data/json/mapgen/small_town_buildings/pet_store.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "pet_store" ],
+    "weight": 100,
+    "object": {
+      "rows": [
+        "ssssssssssssssssssssssss",
+        "ssssssssssssssssssssssss",
+        "_||||www||ss||wwww|||||_",
+        "_|CCW...^|ss|^.....-..|_",
+        "_|CC+....|++|......#..|_",
+        "_|---..............#..|_",
+        "_|CCW..............#..|_",
+        "_|CC+...{..........-D-|_",
+        "_|---...{.............|_",
+        "_|CCW...{.............|_",
+        "_|CCW...{.............|_",
+        "_|CC+...{.............|_",
+        "_|---...{......{{{{{{{|_",
+        "_|CCW...{.....--------|_",
+        "_|CCW.................|_",
+        "_|CC+.................|_",
+        "_||||||||||||||||-...-|_",
+        "___c_________cdd|.....|_",
+        "___c___d___ddadd|....&|_",
+        "___c_ddddddddadd|.....|_",
+        "___c__d__d_d_cddb....&|_",
+        "___c_________cdd|.....|_",
+        "___ccccccccccccc|||||||_",
+        "________________________"
+      ],
+      "terrain": {
+        "#": "t_floor",
+        "&": "t_floor",
+        "+": "t_door_glass_c",
+        "-": "t_wall",
+        ".": "t_floor",
+        "C": "t_concrete",
+        "D": "t_door_c",
+        "W": "t_wall_glass",
+        "^": "t_floor",
+        "_": "t_grass",
+        "a": "t_chaingate_c",
+        "b": "t_door_locked",
+        "c": "t_chainfence_v",
+        "d": "t_dirt",
+        "s": "t_sidewalk",
+        "w": "t_wall_glass_alarm",
+        "{": "t_floor",
+        "|": "t_brick_wall"
+      },
+      "furniture": {
+        "#": "f_counter",
+        "&": "f_sink",
+        "^": "f_indoor_plant",
+        "{": "f_rack"
+      },
+      "place_items": [
+        { "item": "vet_softdrug", "x": 21, "y": 6, "chance": 15 },
+        { "item": "vet_utility", "x": [ 20, 21 ], "y": 3, "chance": 30 }
+      ],
+      "add": [
+        { "item": "catfood", "x": 8, "y": 7 },
+        { "item": "catfood", "x": 8, "y": 8 },
+        { "item": "pet_carrier", "x": 21, "y": 9 },
+        { "item": "dogfood", "x": 8, "y": 10 },
+        { "item": "pet_carrier", "x": 21, "y": 10 },
+        { "item": "dogfood", "x": 8, "y": 11 },
+        { "item": "dogfood", "x": 15, "y": 12 },
+        { "item": "catfood", "x": 16, "y": 12 },
+        { "item": "catfood", "x": 17, "y": 12 },
+        { "item": "pet_carrier", "x": 8, "y": 14 }
```

Comment:
The items should be a group, unless there is a good reason to place exact types.